pages/common/tar.md

```diff
@@ -31,3 +31,7 @@
 - Extract files matching a pattern:
 
 `tar xf {{source.tar}} --wildcards {{"*.html"}}`
+
+- Extract a specific file without the folder structure
+
+`tar xf {{source.tar}} {{source.tar}}/path/to/binary.sh --strip-components={{amount-of-folders-deep}}`
```

Comment:
I don't understand why is `{{source.tar}}/path/to/binary.sh` present here. Anyway, if it stays, it should be put inside `{{}}`.

Reply:
Yes, the whole example should be between the braces.

Reply:
I think the `strip-components` could be explained better. I would say something in the lines of - "Extract a file without creating the given level of directories".

And also, I think just mentioning `--strip-components` is enough. There's no need to extract a single file. The most common usage is to just skip a directory.

Lastly, we use snake_case in our tokens. But we can just shorten it to - {{depth}}